.babelrc

```diff
@@ -0,0 +1,47 @@
+{
+  "env": {
+    "test": {
+      "presets": [
+        [
+          "env",
+          {
+            "targets": {
+              "browsers": ["last 2 versions"]
+            }
+          }
+        ],
+        "stage-0"
+      ]
+    },
+    "development": {
+      "presets": [
+        [
+          "env",
+          {
+            "targets": {
+              "browsers": ["last 2 versions", "ie >= 11"]
+            },
+            "modules": false
+          }
+        ],
+        "stage-0"
+      ],
+      "plugins": ["external-helpers"]
+    },
+    "production": {
+      "presets": [
+        [
+          "env",
+          {
+            "targets": {
+              "browsers": ["last 2 versions", "ie >= 11"]
```

Comment:
Our README currently lists IE 10 as the lowest version we support, but we've tried to make sure we still work with older versions, at least via polyfills. (Because we wanted to try and minimize version restrictions on users of the SDK.)

Old IE versions are now unsupported by MS  — as of [January 12 2016](https://www.microsoft.com/en-us/windowsforbusiness/end-of-ie-support) — but I wonder if we could keep this version at 10 for now until we decide to update our browser support.

Reply:
Fine by me.